src/Slider/Slider.js

```diff
@@ -44,31 +44,60 @@ export default function Slider({
   } else {
     marks = false;
   }
+  // prevent horizontal keyboard navigation for vertical slider
+  const preventHorizontalKeyboardNavigation = (event, orientation) => {
```

Comment:
orientation is never actually passed into preventHorizontalKeyboardNavigation so orientation is always undefined and the keyboard controls are still active. 

I don't think orientation needs to be passed in here as  it doesn't change for the lifetime of the component.